x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_add_edit/steps/step_config.tsx

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiButton, EuiButtonEmpty, EuiFlexGroup, EuiFlexItem, EuiLink } from '@elastic/eui';
+import { FormMonitorType, Step, StepMap } from '../types';
+import { StepFields } from './step_fields';
+
+const MONITOR_TYPE_STEP: Step = {
+  title: i18n.translate('xpack.synthetics.monitorConfig.monitorTypeStep.title', {
+    defaultMessage: 'Select a monitor type',
+  }),
+  children: (
+    <StepFields
+      description={
+        <p>
+          {i18n.translate('xpack.synthetics.monitorConfig.monitorTypeStep.description', {
+            defaultMessage: 'Choose a monitor that best fits your use case',
+          })}
+        </p>
+      }
+      stepKey="step1"
+    />
+  ),
+};
+const MONITOR_DETAILS_STEP: Step = {
+  title: i18n.translate('xpack.synthetics.monitorConfig.monitorDetailsStep.title', {
+    defaultMessage: 'Monitor details',
+  }),
+  children: (
+    <StepFields
+      description={
+        <p>
+          {i18n.translate('xpack.synthetics.monitorConfig.monitorDetailsStep.description', {
+            defaultMessage: 'Provide some details about how your monitor should run',
+          })}
+        </p>
+      }
+      stepKey="step2"
+    />
+  ),
+};
+
+const SCRIPT_RECORDER_BTNS = (
+  <EuiFlexGroup justifyContent="flexStart">
+    <EuiFlexItem grow={false}>
+      <EuiButton href={`elastic-synthetics-recorder://`} iconType="popout" iconSide="right">
+        {i18n.translate('xpack.synthetics.monitorConfig.monitorScriptStep.scriptRecorder.launch', {
+          defaultMessage: 'Launch Script Recorder',
```

Comment:
As far as I know, the official name is "Synthetics Recorder" (see https://github.com/elastic/observability-docs/issues/1907#issuecomment-1169668147). 

```suggestion
          defaultMessage: 'Launch Synthetics Recorder',
```